jobs/utils.py

```diff
@@ -0,0 +1,320 @@
+# Python core Librairies
+import datetime
+import re
+import sqlite3
+
+# Community Librairies
+import pandas as pd
+import requests
+
+from bs4 import BeautifulSoup
+from sqlalchemy import select
+
+
+def check_league(league_name, leagues):
+    '''
+    Check if a league in leagues is in the x, used in a Lambda Function.
+    This is comparing 2 strings between them. Not ideal, but it works.
+    
+    parameters:
+    -----------
+    x: String
+    leagues: list of strings
+    '''
+    for l in leagues:
+        if l in league_name: 
+            return True
+    return False   
+
+def assign_league_id(league_name, leagues_df):
```

Comment:
```suggestion
def assign_league_id(league_name: str, leagues_df: pd.DataFrame) -> None | int | str:
```
typage pour éviter la docstring